cmd/podmanV2/containers/prune.go

```diff
@@ -0,0 +1,84 @@
+package containers
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/cmd/podmanV2/utils"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	pruneDescription = fmt.Sprintf(`podman container prune
+
+	Removes all exited containers`)
+
+	pruneCommand = &cobra.Command{
+		Use:     "prune [flags]",
+		Short:   "Remove all stopped containers",
+		Long:    pruneDescription,
+		RunE:    prune,
+		Example: `podman container prune`,
+	}
+	force  bool
+	filter = []string{}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: pruneCommand,
+		Parent:  containerCmd,
+	})
+	flags := pruneCommand.Flags()
+	flags.BoolVarP(&force, "force", "f", false, "Do not prompt for confirmation.  The default is false")
+	flags.StringArrayVar(&filter, "filter", []string{}, "Provide filter values (e.g. 'label=<key>=<value>')")
+}
+
+func prune(cmd *cobra.Command, args []string) error {
+	var (
+		errs         utils.OutputErrors
+		pruneOptions = entities.ContainerPruneOptions{}
+	)
+	if len(args) > 0 {
+		return errors.Errorf("`%s` takes no arguments", cmd.CommandPath())
+	}
+	if !force {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Println("WARNING! This will remove all stopped containers.")
+		fmt.Print("Are you sure you want to continue? [y/N] ")
+		answer, err := reader.ReadString('\n')
+		if err != nil {
+			return errors.Wrapf(err, "error reading input")
+		}
+		if strings.ToLower(answer)[0] != 'y' {
+			return nil
+		}
+	}
+
+	// TODO Remove once filter refactor is finished and url.Values rules :)
+	for _, f := range filter {
+		t := strings.SplitN(f, "=", 2)
+		pruneOptions.Filters = make(url.Values)
+		pruneOptions.Filters.Add(t[0], t[1])
```

Comment:
Might want to add a check to make sure there are two values contained in `t`.  If not you could get an impressive seg fault.

Reply:
Fixed. Added the check.